src/Symfony/Component/HttpClient/DataCollector/HttpClientDataCollector.php

```diff
@@ -18,8 +18,10 @@
 
 /**
  * @author Jérémy Romey <jeremy@free-agent.fr>
+ *
+ * @final
  */
-final class HttpClientDataCollector extends DataCollector
+class HttpClientDataCollector extends DataCollector
```

Comment:
No need to change this, that's how it is in 4.4, so it should stay as is.

Reply:
My bad, this one was not intentional 😃